mosdepth.nim

```diff
@@ -25,9 +28,32 @@ type
     start: uint32
     stop: uint32
     name: string
+  depth_stat = object
+    cum_length: uint32
+    cum_depth: uint64
+    min_depth: uint32
+    max_depth: uint32
 
   coverage_t = seq[int32]
 
+proc newDepthStat(d: coverage_t): depth_stat =
+  return depth_stat(cum_length: len(d).uint32,
```

Comment:
use int for `cum_length` or we won't be able to handle large choms/genomes.

Reply:
Got it! Thanks!

Reply:
Fixed thanks